go-controller/pkg/node/gateway_udn.go

```diff
@@ -121,10 +121,14 @@ func (b *bridgeConfiguration) delNetworkBridgeConfig(nInfo util.NetInfo) {
 // NOTE: if the network configuration can't be found or if the network is not patched by OVN
 // yet this returns nil.
 func (b *bridgeConfiguration) getActiveNetworkBridgeConfig(nInfo util.NetInfo) *bridgeUDNConfiguration {
+	return b.getActiveNetworkBridgeConfigByName(nInfo.GetNetworkName())
+}
+
+func (b *bridgeConfiguration) getActiveNetworkBridgeConfigByName(network string) *bridgeUDNConfiguration {
```

Comment:
nit: I would prefer not to add another util and instead just change the one other spot in the code :
```
func (c *openflowManager) getActiveNetwork(nInfo util.NetInfo) *bridgeUDNConfiguration {
	return c.defaultBridge.getActiveNetworkBridgeConfig(nInfo)
}
```
to pass the name of the network? looks like we don't really need to pass the whole netInfo anyways?

nit2: if you prefer to keep the ByName util over the other one change the network param to be networkName

Reply:
Makes sense, I changed it to take the networkName instead.